participants/pServer.md

```diff
@@ -0,0 +1,74 @@
+# pServer
+
+## (0) Project Name and Team Slack Handles
+
+- Project Name: **pServer**
+- Filecoin Slack handles:  **arsyun team**, Contact: waynewyang
+
+## (1) Brief Project Description
+
+**pServer is both a demander of ipfs/filecoin and a storage provider of ipfs/filecoin**
+
+pServer is a new product centered on storage service, with expansive function.  pServer is equivalent to a computer without screen, the APP client is equivalent to the remote desktop of the computer without screen, the user will access the pServer through APP client.
+the users of pServer can store and retrieve three types of data.
+
+- 1 Private Data 
+- 2 Sharing data in a small area
+- 3 Sharing data globally(Slingshot), Which store and retrieve in IPFS and filecoin
+
+- Here are some reference information about pserver
+	- [Lightning Demos - pServer in Developers Meetings Berlin July 2018](https://www.youtube.com/watch?v=DGCyGv_1VB8)
+	- [pServer Brief Description](https://arsyun.com/index/enindex.html)
+	- [arsyun github link](https://github.com/arsyun/ARS)
+
+
+## (2) Link to Project UI
+
+- https://arsyun.com/index/endownload.html
+
+## (3) What does your application/UI do?
+
+the users of pServer can store and retrieve three types of data through pServer APP.
+- Store and retrieve private Data
+- Store and retrieve sharing data in a small area
+- Store and retrieve Sharing data globally(Slingshot), Which store and retrieve in IPFS and filecoin
+
+## (4) If your project is using a curated dataset, which one are you using?
+
+- We will give priority to the data in the  [list of curated datasets in this repo](https://github.com/filecoin-project/slingshot/blob/master/datasets.md) for some store and retrieval tests, but if it is slow to obtain the data in the datasets due to network reasons, we will consider switching the test source. Of course, we will submit  PR to the datasets.
+
```

Comment:
Can you be more specific on the starting data set you will use?   The strategy of accessing and indexing the data may be dataset dependent.   As you mentioned, you can always add subsequent data sources but would like to know your starting point.

Reply:
@dineshshenoy We'll use  this datasets firstly. https://github.com/filecoin-project/slingshot/blob/master/datasets.md

Reply:
I think it would still be helpful to identify 1 or 2 as a starting point.